packages/cmrjs/cmr-utils.js

```diff
@@ -277,7 +277,7 @@ async function bucketsConfigDefaults() {
  * @returns {Array<{URL: string, URLDescription: string}>}
  *   returns the list of online access url objects
  */
-function constructOnlineAccessUrls(files, distEndpoint, buckets) {
+function constructOnlineAccessUrls(files, distEndpoint, buckets, s3CredsEndpoint = 's3credentials') {
```

Comment:
I'm not really attached to this implementation. Really what I'm looking for is a decent way to avoid hard-coding in the `s3credentials` in the `cmrjs` package.

This implementation would support stashing the `s3credentials` value in environment variables or in the Cumulus message.

Reply:
I'm not a huge fan, but we're already passing in distEndpoint.  so  seems fine.  Update the js-docstring though.

Reply:
I don't have any opinion about whether this function is the right place to inject the S3 credentials endpoint value.

But in general, any time I see us pushing past 3 arguments to a function, I start to think it would be better to re-factor the code to use an options object instead. It gives us more flexibility now and in the future. Of course, it is less strict from a function signature perspective, which could be a concern.

Either way, this feedback is irrelevant to the actual work for this ticket.